pizone/zone.py

```diff
@@ -88,6 +88,78 @@ def airflow_min(self) -> int:
         """Min allowed airflow for the zone as a percent"""
         return self._get_zone_state('MinAir')
 
+    async def set_airflow_min(self, value: int) -> None:
+        """
+        Change the zone airflow min in 5% increments
+        Async method, returns when server has changed mode.
+        Valid values are percent in 5% increments.
+        Raises:
+            AttributeError if the set point is out of range
+        """
+        if self.type != Zone.Type.OPCL:
```

Comment:
AFAIK, you can set this with all zone types, not just ```Type.OPCL```. Constant zones still have adjustable dampers, and when ```Type.AUTO``` zones run out of batteries on the sensors, they drop back to ```Type.OPCL``` zones.

Reply:
Good catch!